inc/classes/ServiceProvider/class-addons-subscribers.php

```diff
@@ -55,7 +61,18 @@ public function register() {
 		$this->getContainer()->share( 'varnish_subscriber', 'WP_Rocket\Subscriber\Addons\Varnish\VarnishSubscriber' )
 			->withArgument( $this->getContainer()->get( 'varnish' ) )
 			->withArgument( $this->getContainer()->get( 'options' ) );
+		$this->getContainer()->add( 'cloudflare_facade', 'WP_Rocket\Addons\Cloudflare\CloudflareFacade' )
```

Comment:
Any reason not to `->share()` `'cloudflare_facade'` and `'cloudflare'`?

Reply:
These 2 are simple classes which are used as parameters in Cloudflare & CloudflareSubscriber. I don't think it should be shared because are used only inside these classes.

Reply:
Yes `add` is fine here, I use `share` for the subscribers because I want to be able to access the same instance of the class in case we need to remove hooks for example